src/bin/IntaRNA.cpp

```diff
@@ -154,9 +157,31 @@ int main(int argc, char **argv){
 								Predictor * predictor = parameters.getPredictor( *energy, *output );
 								INTARNA_CHECK_NOT_NULL(predictor,"predictor initialization failed");
 
+								// get or compute target regions
+								IndexRangeList targetRegions;
+								const size_t tRegionLenMax = parameters.getTargetRegionLenMax();
+
+								if(tRegionLenMax == 0){
+									targetRegions = parameters.getTargetRanges(targetNumber);
+								}
+								else{
+									targetAcc->compute_accessible_ranges( targetRegions, tRegionLenMax, Accessibility::ED_UPPER_BOUND, 0, targetAcc->getSequence().size());
```

Comment:
seems the `threshold` is the ED upper bound so far and thus only filters for non-accessible sites right?